documentation/API.md

```diff
@@ -124,6 +81,92 @@ Basic login endpoint that will also return (on success):
 }
 ```
 
+#### Get preferences endpoint:
+
+GET: `/get-prefs`
+
+Provides current preferences and language options for the current installation. Called by front-end at the very start of loading in order to configure the environment according to system settings.
+
+#### Get language endpoint:
+
+GET: `/language/<langauge-code>`
+
+Gets the set of localised strings for the core application (i.e. not customisable entities like templates).
+
+#### Verification endpoint
+
+GET: `/verify?uid=<uniqueid>`
+
+Sets the verification record at `<uniqueid>` to `true` and triggers configured Action. Used for verification via email, etc. See [Create Verification Action](List-of-Action-plugins.md) for more info.
+
+#### File download endpoint:
+
+GET: `/file?id=<uniqueId>`
+
+Usage: `GET` request with file database id as a URL query parameter.
+
+This is a public endpoint, but files all have a long uniqueId which should prevent unauthorized access.s
+---
+### Authenticated endpoints
+
+These are all prefixes with `/api`, e.g. `http://localhost:8080/api/login-org`
+
+They require a valid JWT supplied as a Bearer token in the authorization header of the request:
+
+```
+Authorization: Bearer <token>
+```
+
+#### File upload endpoint:
+
+POST: `/upload`
```

Comment:
Nice! Tried without passing the JWT and got this message:
![Screen Shot 2021-10-06 at 3 27 29 PM](https://user-images.githubusercontent.com/16461988/136131209-ccb9e1f6-49e2-4e9c-9709-1c5ae2571a7d.png)